assets/src/components/icons.tsx

```diff
@@ -14,6 +14,7 @@ import StarOutlined from '@ant-design/icons/StarOutlined';
 import UploadOutlined from '@ant-design/icons/UploadOutlined';
 import UpOutlined from '@ant-design/icons/UpOutlined';
 import UserOutlined from '@ant-design/icons/UserOutlined';
+import LogoutOutlined from '@ant-design/icons/LogoutOutlined';
```

Comment:
very minor nitpick -- we try to organize our imports and exports alphabetically if you don't mind :)